modules/efk-logging-curator-delete-index.adoc

```diff
@@ -9,11 +9,11 @@
 This file can be expanded when more functions are supported. Only delete_indices currently. When more functions are added, we can use include::modules/efk-logging-curator-actions.adoc[leveloffset=+1] for a strict delete index files topic, if needed
 ////
 
-You can configure Curator to delete Elasticsearch data.
+Periodically delete Elasticsearch data based on retention settings.
```

Comment:
Here the user isn't actually doing the deleting (like `oc delete pod`), but setting how Curator does the deleting? If so, _You can configure Curator to delete..._ is more accurate. Maybe:
_You can configure Curator to delete Elasticsearch data based on retention settings._